tools/robot_installer.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
Add more comments to this file to group the installation process into steps and explain each step. Put related commands together without lines in between

Reply:
Added comments and removed line spacing.